pkg/osquery/runtime/history/history_test.go

```diff
@@ -189,3 +172,23 @@ func TestLatestInstance(t *testing.T) { // nolint:paralleltest
 		})
 	}
 }
+
+func newTestBoltDb(t *testing.T, instancesToAddToDb ...*Instance) *bbolt.DB {
+	db, err := bbolt.Open(fmt.Sprintf("%s/%s", t.TempDir(), "osquery_instance_history_test.db"), 0600, nil)
+	require.NoError(t, err, "expect no error opening bolt db")
+
+	json, err := json.Marshal(instancesToAddToDb)
+	require.NoError(t, err, "expect no error marshalling instances")
```

Comment:
Maybe reasonable, but it feels slightly weird to load test data here, and not let the `InitHistory` stuff populate it

Reply:
This actually seeds or loads it in the db and lets us confirm that the `InitHistory` stuff loads it without error. I've updated the names and added comment.